### PR TITLE
Docs: Contributing file add

### LINKED ISSUE #20: [DOCS]: Create CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by aayush105**
resolves #20

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## How to Contribute
+
+1. Fork the repository.
+
+2. Clone the forked repository to your local machine.
+
+    ```bash
+    git clone git@github.com:<your_username>/data-communication
+    ```
+
```
Comment (gaurovgiri) [inline CONTRIBUTING.md:16]: Mention about installing the requirements

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Contributing to Data Communication
+
+Thank you for your commitment to supporting Data Communication. Your contribution is vital in creating a collaborative platform that enhances the learning experience for students and researchers. Let's work together to build a resourceful environment where individuals can resolve queries and access the necessary materials for a comprehensive understanding of data communication concepts.
+
+To ensure a smooth and effective contribution process, please review the following guidelines:
+
+## How to Contribute
+
```
Comment (gaurovgiri) [inline CONTRIBUTING.md:8]: Raise an issue or get assigned for an existing issue, if you want to work on something.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    ```bash
+    git checkout -b <branch-name>
+    ```
+4.  Make your changes and commit them.
+
+5.  Push your changes to your forked repository.
+    ```bash
+    git push origin <branch-name>
+    ```
```
Comment (gaurovgiri) [inline CONTRIBUTING.md:27]: Open a pull request

**Review by gaurovgiri — CHANGES_REQUESTED** [review]

**Comment by aayush105** [comment]
@gaurovgiri please review this, changes have been done and push again.

**Review by gaurovgiri — APPROVED** [review]
LGTM!

**Pull request merged**